clarity/src/vm/docs/mod.rs

```diff
@@ -2905,7 +2905,7 @@ mod test {
                 start_height: 0,
                 end_height: u64::MAX,
                 block_limit: ExecutionCost::max_value(),
-                network_epoch: PEER_VERSION_EPOCH_2_1,
+                network_epoch: 0x06,
```

Comment:
Is it possible to keep this as a variable? Otherwise it seems a bit nebulous..

Reply:
Sure